cql2pgjson/src/test/java/org/folio/cql2pgjson/ForeignKeyGenerationTest.java

```diff
@@ -212,4 +212,16 @@ public void searchItemByHoldingsPermanentLocationId() throws Exception {
         + "WHERE permanentLocationId='53cf956f-c1df-410b-8bea-27f712cca7c0')";
     assertEquals(expected, sql);
   }
+
+
+    @Test
+  public void searchItemByHoldingsPermanentLocationName() throws Exception {
+      CQL2PgJSON cql2PgJSON = cql2pgJson("coursereserves_reserves", "courseReservesSchema.json");
+      String sql = cql2PgJSON
+        .toSql("(courseListingId = 1bd04c24-8f8c-4e33-b69c-359146e43808) AND copyrightStatus.name==cc").toString();
```

Comment:
The test should be as concise as possible, `courseListingId = 1bd04c24-8f8c-4e33-b69c-359146e43808` is not needed in the CQL query.

Reply:
fixed